src/bitmapbuffer.cpp

```diff
@@ -351,18 +351,18 @@ static float lb_min(float arr[], int n)
   return m;
 }
 
-bool BitmapBuffer::liangBarskyClipper(coord_t& x1, coord_t& y1, coord_t& x2, coord_t& y2)
+bool BitmapBuffer::liangBarskyClipper(coord_t & x1, coord_t & y1, coord_t & x2, coord_t & y2, coord_t xmin, coord_t xmax, coord_t ymin, coord_t ymax)
```

Comment:
Please avoid unnecessary changes like place of &.

Reply:
this is the style used in libopenui !

Reply:
The “style” is not normalized yet. It will be aligned to the clang-format included in EdgeTX.

Reply:
I guess I am going to give up now ... too much things have added up which are too irrational to me ... I should better concentrate on doing my stuff. Many thx anyways.

Reply:
Does that mean I should finish this myself?

Reply:
up to you guys, you also can close the PR. If you want me to I can close.